release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 [id="ocp-4-16-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-16-installation-and-update-aws-capi_{context}"]
+==== Cluster API replaces Terraform for AWS installations
```

Comment:
Cluster API also replaces Terraform for OpenStack, Nutanix and vSphere. Clearly that is out of scope for this PR, but I'm not sure if it is captured elsewhere.

Reply:
I'll make sure this is settled in a separate PR.